src/main/java/pl/tomaszdziurko/jvm_bloggers/view/panels/CustomPageLinkModel.java

```diff
@@ -0,0 +1,41 @@
+package pl.tomaszdziurko.jvm_bloggers.view.panels;
+
+import lombok.RequiredArgsConstructor;
+
+import org.apache.wicket.markup.html.navigation.paging.IPageable;
+import org.apache.wicket.model.AbstractReadOnlyModel;
+
+@RequiredArgsConstructor
+public class CustomPageLinkModel extends AbstractReadOnlyModel<String> {
```

Comment:
This class looks really vague to me :) Can't we simply modify this css class in `LoopItem` generation code in `CustomPagingNavigator` and maybe extract this logic to separate method? Introducing model with so complicated logic (there are some magic `ifs` with no description that rewrite `idx` value for no clear reason).
It is very difficult to understand what is going on here.